server/routes/mentions.js

```diff
@@ -40,12 +48,15 @@ router.get("/", requiresAuth, async (req, res) => {
           },
         ],
       },
+      order: [["date", "DESC"]],
     });
-
     output = output.concat(mentions);
```

Comment:
in this loop, you're ordering them by date in the getmentions(), and then concatenating them by each company. does this mean that, if two platforms are highlighted, they'll go through all the mentions of one platform before seeing the mentions of the other?

Reply:
I don't think so. Since its ordering by date, and if more than one platform is highlighted, it will just obtain solely on the pagination limit/offset combined with how it was ordered as if it were without pagination